.prettierignore

```diff
@@ -1,4 +1,5 @@
 .cache
 package.json
 package-lock.json
-public
\ No newline at end of file
+public
+gatsby-graphql.ts
```

Comment:
Auto generated file. No need to format it since we are not going to change it manually.